packages/wrangler/templates/middleware/middleware-d1-beta.ts

```diff
@@ -1,13 +1,11 @@
-// src/shim.ts
-import worker, * as OTHER_EXPORTS from "__ENTRY_POINT__";
-export * from "__ENTRY_POINT__";
+/// <reference path="middleware-d1-beta.d.ts"/>
```

Comment:
can we hold off from porting the d1 shim? we're planning on moving it into workerd very very shortly, so we'll only have two types of users: folks with workers w/ the old shim applied, and folks that have a native d1 shim

Reply:
after talking to @petebacondarwin about this, I agree this change should be fine - could we just base it on top of main once https://github.com/cloudflare/workers-sdk/pull/3414 merges?